ui/packages/tidb-dashboard-lib/src/apps/Statement/pages/Detail/PlanBind.tsx

```diff
@@ -0,0 +1,292 @@
+import React, { useContext, useState, useMemo, useRef, useEffect } from 'react'
+import { Space, Button, Modal, Tooltip, Radio, Alert } from 'antd'
+import { InfoCircleOutlined } from '@ant-design/icons'
+
+import { useClientRequest } from '@lib/utils/useClientRequest'
+import { StatementModel } from '@lib/client'
+import { useTranslation } from 'react-i18next'
+import { IPageQuery } from '.'
+import { StatementContext } from '../../context'
+import { CardTable } from '@lib/components'
+import styles from './PlanBind.module.less'
+import { planColumns as genPlanColumns } from '../../utils/tableColumns'
+import {
+  SelectionMode,
+  CheckboxVisibility
+} from 'office-ui-fabric-react/lib/DetailsList'
+import { Selection } from 'office-ui-fabric-react/lib/Selection'
+
+interface PlanBindProps {
+  query: IPageQuery
+  plans: StatementModel[]
+}
+
+const PlanBind = ({ query, plans }: PlanBindProps) => {
+  const ctx = useContext(StatementContext)
+  const { t } = useTranslation()
+  const { data: planBindingStatus } = useClientRequest((reqConfig) =>
+    ctx!.ds.statementsPlanBindStatusGet!(
+      query.digest!,
+      query.beginTime!,
+      query.endTime!,
+      reqConfig
+    )
+  )
+
+  const [hasPlanToBind, setHasPlanToBind] = useState(false)
+  const [boundPlanDigest, setBoundPlanDigest] = useState<string | null>(null)
+  const [showPlanBindModal, setShowPlanBindModal] = useState(false)
+
+  useEffect(() => {
+    if (planBindingStatus) {
+      setBoundPlanDigest(planBindingStatus.plan_digest!)
+    }
+  }, [planBindingStatus])
+
+  useEffect(() => {
+    plans.forEach((plan) => {
+      if (plan.plan_digest) {
+        setHasPlanToBind(true)
+      }
+    })
+  }, [plans])
+
+  return (
+    <Space align="center">
+      {!hasPlanToBind ? (
+        <Tooltip
+          title={t('statement.pages.detail.plan_bind.bound_available_tooltip')}
+          placement="leftTop"
+        >
+          <InfoCircleOutlined /> Unavailable
+        </Tooltip>
+      ) : (
+        <>
```

Comment:
I guess this `<>` is redundant.

Reply:
We need this `<>` since there is an another following Conditional operator.